pom.xml

```diff
@@ -160,10 +195,23 @@
         <debezium-connector-mysql.version>0.8.3.Final</debezium-connector-mysql.version>
         <debezium-embedded.version>0.8.3.Final</debezium-embedded.version>
         <debezium-connector-oracle.version>0.8.3.Final</debezium-connector-oracle.version>
+        <hikari.version>3.2.0</hikari.version>
         <org.testng.version>6.11</org.testng.version>
         <siddhi-store-rdbms.version>4.0.39</siddhi-store-rdbms.version>
+        <carbon.datasource.core.version>RELEASE</carbon.datasource.core.version>
+        <mavan.findbugsplugin.exclude.file>../findbugs-exclude.xml</mavan.findbugsplugin.exclude.file>
+        <ojdbc6.version>12.1.0.1-atlassian-hosted</ojdbc6.version>
+        <sqljdbc4.version>4.0</sqljdbc4.version>
+        <postgresql.version>RELEASE</postgresql.version>
+        <h2.connector.version>1.4.187</h2.connector.version>
         <siddhi-cdc-mysql.port>3309</siddhi-cdc-mysql.port>
         <docker.container.siddhi-cdc-mysql.ip>0.0.0.0</docker.container.siddhi-cdc-mysql.ip>
+        <siddhi-cdc-oracle.port>1522</siddhi-cdc-oracle.port>
```

Comment:
If these are related to plugin arguments, then move inside config tag of each plugin call

Reply:
variables like `siddhi-cdc-oracle.port` are defined in the parent pom because they are being used inside more than one plugin.